### DISCUSSION TITLE
Error: Invalid JSON in the IfElse's Input Variables: SyntaxError: Expected ',' or '}' after property value in JSON at position 15

### DISCUSSION THREAD

**Discussion by Karthikeyakucham6918** [FlowiseAI/Flowise#2573] [Q&A]
I have used structured output parser to pass the output as json into the if else condition utility then I am facing the above error, do any one faced this type of issue?..

**Comment by NoxGnosis92** [comment]
I’m getting the same issue. I remove a couple of the brackets, and it functions, but it always reads “else” now.

**Reply by Adnandeardigital** [reply]
Yes, you are correct. I was also able to remove the bracket issue, but it always read 'else'. So, I worked around it by using an integer instead of a boolean, which gave me the correct results.

![image](https://github.com/FlowiseAI/Flowise/assets/106959148/6dc6a353-a80f-45db-886d-27dd738d2263)
![image](https://github.com/FlowiseAI/Flowise/assets/106959148/17ed26a4-30e0-4acd-b1e2-2c3b24c74869)
![image](https://github.com/FlowiseAI/Flowise/assets/106959148/664ef0a6-a1da-414b-bde7-b6af0876edbe)

**Comment by HenryHengZJ** [comment]
I think it was due to this bug here - https://github.com/FlowiseAI/Flowise/pull/2577, will be pushing to new release soon

**Comment by Adnandeardigital** [comment]
I was following this youtube tutorial https://www.youtube.com/watch?v=bT308854-Qw&t=143s
and receieved same error was anyone able to fix it ? 

_Error: Invalid JSON in the IfElse's Input Variables: SyntaxError: Expected ',' or '}' after property value in JSON at position 11_
